### ISSUE TITLE
IE 11 

### ISSUE THREAD

**Issue by maenabbasi** [google/google-visualization-issues#2243]
I got the following Error on IE
 SCRIPT380: Invalid property value.

**Comment by dlaliberte** [comment]
Can you give us some more clues about how you got this error?  Point us at an example, if you can.

**Comment by maenabbasi** [comment]
using the following code on IE

**Comment by maenabbasi** [comment]
kindly find the attached file

**Comment by maenabbasi** [comment]
[huge_data.txt](https://github.com/google/google-visualization-issues/files/217407/huge_data.txt)

**Comment by maenabbasi** [comment]
[index.txt](https://github.com/google/google-visualization-issues/files/217416/index.txt)

**Comment by dlaliberte** [comment]
OK thanks.  The orgchart is one that we have not tried to update much.  I don't see offhand what property might be causing a problem in IE, but if you can narrow it down, that would be helpful.  Try with a smaller set of data first.

**Comment by maenabbasi** [comment]
Yes it works with small data set, but i want it for data 1500records and
highers

On Wednesday, 13 April 2016, Daniel LaLiberte notifications@github.com
wrote:

> OK thanks. The orgchart is one that we have not tried to update much. I
> don't see offhand what property might be causing a problem in IE, but if
> you can narrow it down, that would be helpful. Try with a smaller set of
> data first.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/google/google-visualization-issues/issues/2243#issuecomment-209517572

## 

Sent from Gmail Mobile

**Comment by jcw-** [comment]
Duplicate of #168
